src/components/InputDropdown/styles.ts

```diff
@@ -0,0 +1,143 @@
+import styled, { keyframes } from 'styled-components';
```

Comment:
safari problem 4379423: for some reason the input dropdown glitches every time i open it up,,,

it works completely fine so its not a huge deal,,, but just wanted to flag!

https://github.com/calblueprint/immigration-justice-project/assets/47801900/ca60e2c7-b682-4a9e-8066-d7f1bc5bae7a

Reply:
oh that might be the animation.. :/

Reply:
ill try to remove the z-index in the animation and add a `forwards` keyword, let me know if anything changes!

Reply:
looks good now! ty jinkang